### DISCUSSION TITLE
Definir mecánicas del jugador

### DISCUSSION THREAD

**Discussion by LeoMast** [gamerevar/proyecto-comunitario-1-unity#13] [Game Design]
Abro este hilo para definir un par mecánicas que se hablaron en discord, la idea poder avanzar lo mas democráticamente posible. Les pido Voten por las opciones que mas le convencen.

**Lo definido:**
 El jugador mueve al protagonista con wasd y posee un Dron que lo sigue a todas partes.

**A definir:** 
**1 Como sigue el Dron al jugador?**

A.   El Dron sigue  con una pequeña demora al jugador (sin girar a su alrededor) y dispara hacia donde apunta el mouse.
B.  El Dron sigue  con una pequeña demora al jugador (sin girar a su alrededor) y dispara hacia donde mira el jugador pero con el mouse se pude disparar  en una dirección dentro del campo de visión del jugador.
C. El Dron gira al rededor del jugador y dispara hacia donde apunta el mouse. 

**2 Como ataca el Dron?** 

A.  El Dron posee 2 armas, la primera es un laser, la segunda otro tipo de arma que  realiza daño de área ( que se puede desbloquear después de x nivel)
B. EL Dron posee 2 armas, la primera es un rifle ( o algo similar), la segunda es una escopeta ( o dispara como si fuera) ( también se desbloquea en x nivel)
C. El Dron solo  tiene un arma, y mejora ganando xp. ( el tema mejoras nos sacaría del scope )

**3  el jugador tiene arma?**

Considero que el jugador debe tener un arma para pelear cuerpo a cuerpo, 

A. Cuchillo o Machete
B. Martillo tamaño mediano, 
C. Sin arma  
 D. Sin arma pero el jugador tiene un dash para evadir a los enemigos cuando están demasiado cerca. 

En mi caso prefiero ; 
**1.A**  El Dron sigue  con una pequeña demora al jugador (sin girar a su alrededor) y dispara hacia donde apunta el mouse.
**2.A**El Dron posee 2 armas, la primera es un laser, la segunda otro tipo de arma que  realiza daño de área ( que se puede desbloquear después de x nivel)
y
**3.A** Cuchillo o Machete  sumandole el Dash para evadir enemigos

**Comment by Swimmablebee912** [comment]
No soy de gamedesing jajaja, pero, yo prefiero estas:
1_A para dejarle la dificultad enteramente a la skill del jugador
2_B para darle opciones similares, pero, que funcionen diferente
3_D asi lo dejariamos 100% dependiente del dron

**Reply by LeoMast** [reply]
perdón pero no explique bien la segunda arma, 1 es un laser y la segunda es otra diferente que hace daño de área. ya lo cambio.
Gracias por aportar, si la idea es que aporten todos los que quieran.
Y me gusta esa idea de que el jugador  sude jeje.

**Reply by AgustinAllamanoCosta** [reply]
Esta es mi eleccion
1_A 
2_A
3_D

**Reply by NNmac** [reply]
1. A
2. B
3. D

**Comment by Svartskogen** [comment]
1. A
2. A o B
3. D
